include/nuttx/list.h

```diff
@@ -79,6 +79,15 @@
 #define list_entry(ptr, type, member) \
            container_of(ptr, type, member)
 
+/**
+ * list_entry_is_head - test if the entry points to the head of the list
+ * @pos:    the type * to cursor
+ * @head:   the head for your list.
+ * @member: the name of the list_head within the struct.
+ */
+#define list_entry_is_head(pos, head, member) \
+            (&pos->member == (head))
+
```

Comment:
Besides been copied from Linux, these comments are not following the NuttX coding style